.github/workflows/haskell.yml

```diff
@@ -400,7 +400,7 @@ jobs:
       SECP256K1_REF: ac83be33d0956faf6b7f61a60ab524ef7d6a473a
 
       SECP_CACHE_VERSION: "2022-12-30"
-      ghc-version: "9.10.1"
+      ghc-version: "9.6.7"
```

Comment:
How come this change?  Any particular reason?

Reply:
Ah, I see! I think it's a restriction coming from plutus-tx-plugin, but  I don't know either why. 
Thanks!

Reply:
> I think it's a restriction coming from plutus-tx-plugin, but I don't know either why.

plutus-tx-plugin can only support one major ghc version at a time, since they rely heavily on ghc internals, so that is why it is only buildable with ghc-9.6